README.md

```diff
@@ -25,47 +27,40 @@ First of all, clone the github repository: `git clone https://github.com/project
 Simply run
 
 ```
-pip install .
-```
-
-If you want to do some changes to the code, I suggest to do:
-
-```
-cd atomicapp
-export PYTHONPATH=`pwd`:$PYTHONPATH
-alias atomicapp="python `pwd`/atomicapp/cli/main.py"
```

Comment:
i don't think this should be included ^^. let python devs do their own way of development